### ISSUE TITLE
Why it not support IOS ? 

### ISSUE THREAD

**Issue by phuongphally** [NPKompleet/Beatz#1]

**Comment by NPKompleet** [comment]
Hi @phuongphally I am currently focusing more on Android for now because I am way more skilled at Android than iOS. But once I am satisfied with the Android version, I can start thinking of supporting iOS.

**Comment by phuongphally** [comment]
We move from Java to dart because multiple platform support .

**Comment by NPKompleet** [comment]
Yeah. The major advantage of Flutter is multiple platform support. But like I said I am focussing on getting it right in Android first.

**Comment by phuongphally** [comment]
@NPKompleet  Thanks for reply
